README.md

```diff
@@ -97,6 +97,7 @@ GoogleSignin.configure({
   webClientId: '<FROM DEVELOPER CONSOLE>', // client ID of type WEB for your server (needed to verify user ID and offline access)
   offlineAccess: true, // if you want to access Google API on behalf of the user FROM YOUR SERVER
   hostedDomain: '', // specifies a hosted domain restriction
+  loginHint: '', // [iOS] specifies an email address or user ID to show when presenting the login screen
```

Comment:
thanks for the contribution! The docs at https://developers.google.com/identity/sign-in/ios/api/interface_g_i_d_sign_in say that 

> The login hint to the authorization server, for example the user's ID, or email address, to be prefilled if possible.

So, is this something that is being shown to the user or is it more of a server thing? 

you saying:

>  specifies an email address or user ID to show when presenting the login screen

can you just add a screenshot of where this can be found? Thanks!

Reply:
Sure! This usually causes the hinted account address (ie. foo@bar.com) to be pre-filled in the sign-in screen that is presented to the user (see screenshot below), though if the hinted user is already authenticated, this can cause authentication to happen without user interaction.
<img width="598" alt="screen shot 2018-11-05 at 3 52 23 pm" src="https://user-images.githubusercontent.com/651917/48029933-836c1580-e11d-11e8-97a2-2e732450d0e6.png">

Reply:
@cgilboy 
> though if the hinted user is already authenticated, this can cause the react-native-google-signin flow to happen without user interaction.

okay, that's an interesting piece of information, can you add this and potentially other things you found out to the docs (appending that behind what you already added is okay) and / or add a link to the part of the official docs that covers this? Thanks

Reply:
I've updated the README text and added a link to Google's docs. I don't feel comfortable including my observation about the automatic authentication because it is undocumented and I don't know what current/future expected behavior of Google's library is.

Reply:
Follow-up question - is there any sense in combining the options for iOS's `loginHint` with Android's `accountName`? https://developers.google.com/android/reference/com/google/android/gms/auth/api/signin/GoogleSignInOptions.Builder.html#setAccountName(java.lang.String)
I'm not very familiar with the behavior on Android, but it sounds like it has a similar intention, though only based on account name, not User ID.

Reply:
@vonovak Can do. Should we specify that it can be a User ID on iOS?
```suggestion
accountHint: '', // specifies an account to preselect in the authorization UI. On Android, must be an account name. On iOS, can be an email address or a user's ID.
```